examples/sycl/decision_forest_classification_batch.py

```diff
@@ -0,0 +1,143 @@
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License"), the following terms apply:
+#
+# You may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*******************************************************************************
+
+# daal4py Decision Forest Classification example for shared memory systems
+
+import daal4py as d4p
+import numpy as np
+import os
+from daal4py.oneapi import sycl_buffer
+
+# let's try to use pandas' fast csv reader
+try:
+    import pandas
+    read_csv = lambda f, c, t=np.float64: pandas.read_csv(f, usecols=c, delimiter=',', header=None, dtype=t)
+except Exception as e:
+    # fall back to numpy loadtxt
+    read_csv = lambda f, c, t=np.float64: np.loadtxt(f, usecols=c, delimiter=',', ndmin=2, dtype=t)
+
+# Get Intel(R) Data Analytics Acceleration Library (Intel(R) DAAL) version
+from daal4py import __daal_link_version__ as dv
+daal_version = tuple(map(int, (dv[0:4], dv[4:8])))
+
+try:
+    from dpctx import device_context, device_type
+    with device_context(device_type.gpu, 0):
+        gpu_available=True
+except Exception as e:
+    try:
+        from daal4py.oneapi import sycl_context
+        with sycl_context('gpu'):
+            gpu_available=True
+    except Exception as e:
+        gpu_available=False
+
+# Commone code for both CPU and GPU computations
+def compute(train_data, train_labels, predict_data, method='defaultDense'):
+    # Configure a training object (5 classes)
+    train_algo = d4p.decision_forest_classification_training(5,
+                                                             nTrees=10,
+                                                             minObservationsInLeafNode=8,
+                                                             featuresPerNode=3,
+                                                             engine = d4p.engines_mt19937(seed=777),
+                                                             varImportance='MDI',
+                                                             bootstrap=True,
+                                                             resultsToCompute='computeOutOfBagError',
+                                                             method=method
+                                                             )
+    # Training result provides (depending on parameters) model, outOfBagError, outOfBagErrorPerObservation and/or variableImportance
+    train_result = train_algo.compute(train_data, train_labels)
+
+    # now predict using the model from the training above
+    if daal_version < (2020,1):
```

Comment:
Need delete it. We save version check for previous release

Reply:
thanks, removed